### ISSUE TITLE
Can html imports be minimized by the jspm bundler?

### ISSUE THREAD

**Issue by stumblingmages** [systemjs/plugin-text#30]
Is it possible to import an html template (this is an angular app) and have that import minimized when the bundle-sfx is run? Or is it necessary to minify the html somewhere earlier in the build process?

**Comment by guybedford** [comment]
This is certainly possible through custom template plugins that can do the compression on build, but not a feature of this plugin.

**Comment by stumblingmages** [comment]
I have a sort of workable solution using a clone of the text plugin for html. I hooked into the fetch and worked with the load.address field to check for a minified version of the template and return that if it existed. If it didn't I just returned the file specified by load.address. This was done with the jspm installed fs and readFileSync functions. Besides the sync functions, I also need to replace parts of the load.address string to look in build location first. Not ideal at all but it does work.

If there's a way I can configure my plugin with match and replace text, it would be universally applicable. Used like `load.address.replace(/main\/webapp/, 'build/templates')`

Also tried using the html-minifier from within the transport function to minify my html on the fly without need to build a minified version of the file. This would be optimal as it would make my plugin universal but could not get the html-minifier to load.
